lesson_08/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: "1"
+services:
+    saver:    
+        build: .\saver\
```

Comment:
That path is not valid for me, @denislituev is that the same for you?
I propose to use `saver/`

Reply:
I think it should be` saver/`